src/pandablocks_ioc/_pvi.py

```diff
@@ -61,9 +63,16 @@ def add_pvi_info(
     useComboBox: bool = record_creation_func == builder.mbbOut
 
     if record_creation_func == builder.Action:
-        # TODO: What value do I write? PandA uses an empty string
-        component = SignalX(record_name, record_name, value="")
-        access = "x"
+        if record_name == "PCAP:ARM":
```

Comment:
I investigated checking the record for `ONAM: "Arm"` `ZNAM: "Disarm"` but with a `RecordWrapper` this code would be harder to read and have worse typing than simply checking if `record_name == "PCAP:ARM"'